packages/runtime/src/runtime.ts

```diff
@@ -388,6 +388,28 @@ export class Runtime {
     toAccount.amount += txnParam.amountMicroAlgos; // add 'x' algo to receiver
   }
 
+  /**
+   * validate logic signature and teal logic
+   * @param txnParam Transaction Parameters
+   */
+  validateLsig (txnParam: ExecParams): void {
```

Comment:
Can we change the function to `validateLsigAndRun`? We are executing TEAL code along with validating lsig in this function. Or we can `run` the code after validation

Reply:
validate lsig means we are checking logic too, but yes i can update